INSTALL.adoc

```diff
@@ -2,22 +2,33 @@
 
 == Prerequisites
 
-* The GNU C Compiler (gcc) is recommended, as the bytecode interpreter takes
+* A C Compiler is required.
+  The GNU C Compiler (`gcc`) is recommended as the bytecode interpreter takes
   advantage of GCC-specific features to enhance performance. gcc is the standard
-  compiler under Linux, OS X, and many other systems.
+  compiler under Linux, and many other systems.
+  However `clang` used in Mac OS, BSDs and others, also works fine.
+
+* GNU `make`, as well as GNU `awk` and GNU `sed` are required.
+  Alternatively, Dune can also be used but the support is experimental.
```

Comment:
Agreed: I'm not sure the Dune build can build the whole system at the moment, so please don't advertise for it.  Plus, as far as I can remember, it could call into makefiles to build the runtime system, including sed and awk invocations.